### ISSUE TITLE
How to add custom behaviour on click event to sunburst component built with react-kapsule?

### ISSUE THREAD

**Issue by RajS999** [vasturiano/react-kapsule#11]
When [sunburst component](https://github.com/vasturiano/sunburst-chart) is used in react built with `react-kapsule`, how can I add custom behavior on click event of any slice of sunburst. 

From suggestion [here](https://github.com/vasturiano/sunburst-chart/issues/40#issuecomment-533369204), I tried something like this:
```
  <ReactSunburst
    //...
    data={data}
    onClick={(node) => {
      myChart.focusOnNode(node);
      console.log("clicked");
    }}
  /> 
```

The problem I faced is that `myChart` is undefined inside the `onClick` handler of react component. So, this gives `myChart is not defined` error and the chart does not perform the default zooming behavior. If I omit the first line, "clicked" gets printed to the console.
[Codesandbox link](https://codesandbox.io/s/sunburst-test-forked-73kye4?file=/src/components/SunburstChart.js:501-533).

**Comment by RajS999** [comment]
Sorry, I just realized that [we have to add the methods to `methodNames` array](https://github.com/vasturiano/react-kapsule#arguments). So I tried this:

```
const ReactSunburst = fromKapsule(SunburstChart, {
  methodNames: ["onClick"]
});

<ReactSunburst
    width={200}
    height={200}
    excludeRoot
    label="name"
    size="size"
    color={(d, parent) => null}
    tooltipContent={(d, node) => `Size: <i>${node.value}</i>`}
    data={flare}
    onClick={(entry) => {
      console.log("Hello from inside onClick handler!!");
    }}
  />
```

Now clicking on slices zooms in as desired. However, it does not seem to invoke the `onClick` handler at all as the message `Hello from inside onClick handler!!` is not printed to the console. Here is the [link to this codesandbox](https://codesandbox.io/s/xenodochial-kalam-b3lpj4-shared-b3lpj4?file=/src/Chart.js:201-225). 

And when I we omit `methodNames`, the message gets printed, but the slice does not zooms in. Here is the [link to sandbox with this change](https://codesandbox.io/s/xenodochial-kalam-b3lpj4-no-method-shared-z7her5?file=/src/Chart.js:201-216).

It will be of great help to find one working example of react-capsule + sunburst-chart with click event handler.
